src/common/common.ts

```diff
@@ -28,4 +28,5 @@ export type StoreStructure = {
   };
   playlists: Playlist[];
   lastPlayedSong: string; // a key into "library"
+  libraryPath: string;
```

Comment:
makes it foolproof when we want to copy newly added songs into the library folder.